.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+on: [push]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.17.x
+      - name: Yarn install
+        uses: borales/actions-yarn@v4
+        with:
+          cmd: install
+      - run: yarn check
+      - run: yarn lint
+      - run: yarn flow
+      - run: yarn compile
+      - run: yarn test
+      - run: yarn dist
+      - run: yarn riffraff
```

Comment:
This presumably works fine, but wouldn't the "Upload to riff-raff" step below, achieve the same thing? Is this still necessary? Could remove that package and this line?

Reply:
Agreed. We should do one of these, not both. Doing both means we're uploading the same artifacts to Riff-Raff twice, which is unnecessary.

Instead, we should use `guardian/actions-riff-raff` and remove the [deprecated NPM module](https://www.npmjs.com/package/@guardian/node-riffraff-artifact).